                    Применение нескольких операторов catch.

    Иногда в одном фрагменте кода может возникнуть не одно исключение. Чтобы справиться
с такой ситуацией, можно указать два или больше оператора catch, каждый из которых пред-
назначается для перехвата отдельного типа исключения. Когда генерируется исключение, каж-
дый оператор catch проверяется по порядку и выполняется тот из них, который совпадает по
типу с возникшим исключением. По завершении одного из операторов catch все остальные про-
пускаются и выполнение программы продолжается с оператора, следующего сразу после блока
операторов try/catch.

Пример 1:
    В этой программе произойдет исключение в связи с делением на нуль, если она будет
запущена без аргументов командной строки. Ведь в этом случае значе­ние переменной а будет
равно нулю. Деление будет выполнено нормально, если программе будет передан аргумент коман-
дной строки, устанавливающий в пере­менной а значение больше нуля. Но в этом случае возникнет
исключение типа ArrayIndexOutOfBoundsException, поскольку длина массива целых чисел с равна 1,
тогда как программа пытается присвоить значение элементу массива с[42].


